js/app/modules/view/document.js

```diff
@@ -19,28 +18,27 @@ const PDFView = imports.app.widgets.PDFView;
 const SlidingPanelOverlay = imports.app.widgets.slidingPanelOverlay;
 const TableOfContents = imports.app.widgets.tableOfContents;
 const Utils = imports.app.utils;
+const {View} = imports.app.interfaces.view;
```

Comment:
I hope to move to this syntax more widely in new code. so I don't know if it's worth leaving a comment in this particular instance; but it's [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment). Basically means take the `View` member of the RHS and assign it to the `View` local variable. It avoids having to refer to `View.View` everywhere as we did with `Card.Card` for example, and adheres more closely to the upcoming `import {View} from "app/interfaces/view"` syntax for ES6 modules.